instrumentation/servlet/servlet-3.0/javaagent/src/test/groovy/TestServlet3.groovy

```diff
@@ -173,7 +173,6 @@ class TestServlet3 {
               case HTML2:
                 resp.contentType = "text/html"
                 resp.status = endpoint.status
-                resp.setContentLengthLong(endpoint.body.length())
```

Comment:
is this removed to test the case where content length wasn't set? (makes sense just checking that it wasn't accidentally removed as part of debugging)